app/src/main/res/layout/activity_scheduled_toot.xml

```diff
@@ -23,6 +23,18 @@
             app:layout_constraintRight_toRightOf="parent"
             app:layout_constraintTop_toTopOf="parent" />
 
+        <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
```

Comment:
why did it move?

Reply:
because the button of the error view is not clickable if the `SwipeRefreshLayout` is above it